planter/planter.sh

```diff
@@ -42,8 +42,8 @@ set -o nounset
 # these can be overridden but otherwise default to the current stable image
 # used to build kubernetes from the master branch
 IMAGE_NAME="${IMAGE_NAME:-gcr.io/k8s-testimages/planter}"
-TAG="${TAG:-0.21.0}"
-IMAGE=${IMAGE:-${IMAGE_NAME}:${TAG}}
+TAG="${TAG:-0.22.0}"
+IMAGE=${IMAGE:-${IMAGE_NAME}:${TAG}${CROSS+-cross}}
```

Comment:
_bash intensifies_

Reply:
we should update the README and then environment variable options block above with how to set this.

Reply:
good call, done.